lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -7,9 +7,15 @@ use MusicBrainz::Server::Translation qw( l ln );
 use MusicBrainz::Server::Validation qw( is_valid_url );
 
 extends 'MusicBrainz::Server::Form';
+with 'MusicBrainz::Server::Form::Role::ValidateUsername';
 
 has '+name' => ( default => 'profile' );
 
+has_field 'username' => (
+    type      => 'Text',
+    maxlength => 64
```

Comment:
Should have a trailing comma, so that adding more entries later doesn’t require changing this line.